src/main/java/seedu/address/logic/commands/ListStudentsCommand.java

```diff
@@ -0,0 +1,25 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.model.Model.PREDICATE_SHOW_ALL_PERSONS;
+
+import seedu.address.model.Model;
+
+/**
+ * Lists all persons in the address book to the user.
+ */
+public class ListStudentsCommand extends Command {
+
+  public static final String COMMAND_WORD = "list_students";
```

Comment:
I think according to our current design and documents, commands should have a '/' in front of them i.e. '/list_student'.